### PR TITLE
#835 - upload install test

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #835
Added test case for upload and install

### CODE REVIEW

File: src/test/java/com/artipie/debian/DebianITCase.java [hunk 1]
```diff
@@ -81,21 +88,26 @@
...
         final Path setting = this.tmp.resolve("sources.list");
         Files.write(
             setting,
             String.format(
                 "deb [trusted=yes] http://host.testcontainers.internal:%d/my-debian %s main",
-                port, DebianITCase.NAME
+                this.port, DebianITCase.NAME
             ).getBytes()
         );
         this.cntn = new GenericContainer<>("debian")
```
Comment (genryxy) [inline src/test/java/com/artipie/debian/DebianITCase.java:113]: @olenagerasimova I'd suggest to use `TestContainer` class here

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#837` is not assigned, can't get performer

**Pull request merged**